net-mgmt/collectd/src/opnsense/mvc/app/models/OPNsense/collectd/Menu/Menu.xml

```diff
@@ -0,0 +1,5 @@
+<menu>
+  <Services>
+    <Collectd cssClass="fa fa-address-book-o" url="/ui/collectd/general/index" />
```

Comment:
would prefer fa-area-chart as it looks more like how it is displayed here using rrdtool (https://collectd.org/)